### PR TITLE
VCR recordings for ARM azure_mgmt_resources tests

### PR DESCRIPTION
**Pull request by vishrutshah**
Adding support for VCR recording in ARM azure_mgmt_resources service

This changes is part of the feature #291

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vishrutshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: resource_management/azure_mgmt_resources/spec/tags_spec.rb [hunk 1]
```diff
@@ -8,47 +8,45 @@
...
+    expect(result.response.status).to eq(200)
   end
 
-  it 'should create tag' do
-    name = get_random_name('RubySDKTest_')
-    @created_tags.push(name)
-
-    result = @client.create_or_update(name).value!
+  it 'should create and delete tag' do
+    result = @client.create_or_update(tag_name).value!
```
Comment (veronicagg) [inline resource_management/azure_mgmt_resources/spec/tags_spec.rb]: is there a reason to not include this line in the 'before' block?

**Comment by veronicagg** [comment]
@vishrutshah Left one comment, rest looks good to me! Thanks.

File: resource_management/azure_mgmt_resources/spec/tags_spec.rb (hunk 1 above)
Comment (vishrutshah) [inline resource_management/azure_mgmt_resources/spec/tags_spec.rb]: Nope. I should move it.

**Comment by veronicagg** [comment]
@vishrutshah Looks good, thanks!

**Comment by vishrutshah** [comment]
Thanks for the review !!

**Pull request merged**